test/parallel/test-assert-fail.js

```diff
@@ -23,7 +23,7 @@ assert.throws(
     code: 'ERR_ASSERTION',
     type: assert.AssertionError,
     operator: undefined,
-    actual: undefined,
+    actual: 'custom message',
```

Comment:
Okay, so it sounds like everything is working as expected and better than before? :)